index.js

```diff
@@ -56,7 +56,8 @@ export const Rels = {
 		previousWeek: 'https://activities.api.brightspace.com/rels/previous-week',
 		feedback: 'https://activities.api.brightspace.com/rels/feedback',
 		activityGrade: 'https://activities.api.brightspace.com/rels/activity-grade',
-		activityHomepage: 'https://activities.api.brightspace.com/rels/activity-homepage'
+		activityHomepage: 'https://activities.api.brightspace.com/rels/activity-homepage',
+		evaluation: 'https://api.brightspace.com/rels/evaluation'
```

Comment:
I think this needs modification in one direction or another - generally, if a rel is just on the `api.brightspace.com` domain, we keep it in the top level here, to match that. (Things in the `Activities` object here are all `activities.api.brightspace.com`.) Alternatively, if evaluations are strictly an activities-domain thing, then this should probably be modified LMS-side to be on the `activities.api.brightspace.com` domain.

Reply:
Thanks I did not notice that, I will have another look

Reply:
After some discussion, decided to put this up to top level https://git.dev.d2l/projects/CORE/repos/lp/pull-requests/12732/overview?commentId=339523